packages/typeorm-decorator-ismoney/src/index.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  registerDecorator,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+} from 'class-validator';
+
+import codes from './codes';
+
+@ValidatorConstraint()
+export class IsMoneyValidator implements ValidatorConstraintInterface {
+  public validate(money: any) {
+    const { amount, currency } = money;
+    const isAmountCorrect =
+      typeof amount === 'number' && amount >= 0 && amount <= 999999999;
```

Comment:
what is 999999999 ?

Reply:
> what is 999999999 ?
I've taken it from gateway isMoney validation. On the second thought, there should be optional margins for the amount.

Reply:
we need to check commit message why it was added

Reply:
> Limit max money amount (#374)